chainforge/react-server/src/ModelSettingSchemas.js

```diff
@@ -254,7 +254,7 @@ const ClaudeSettings = {
                 "type": "string",
                 "title": "Model Version",
                 "description": "Select a version of Claude to query. For more details on the differences, see the Anthropic API documentation.",
-                "enum": ["claude-2", "claude-2.0", "claude-2.1", "claude-instant-1", "claude-instant-1.1", "claude-instant-1.2", "claude-v1", "claude-v1-100k", "claude-instant-v1", "claude-instant-v1-100k", "claude-v1.3", 
+                "enum": ["claude-2", "claude-2.0", "claude-2.1", "claude-instant-1", "claude-instant-1.1", "claude-instant-1.2", "claude-v1", "claude-v1-100k", "claude-instant-v1", "claude-instant-v1-100k", "claude-v1.3",
```

Comment:
Looks like your editor doesn't trim newlines on save, while my editor does. I added these changes but I can remove them if you prefer.